composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "a1comms/gae-support-laravel",
```

Comment:
We also won't be able to merge until this is removed.

Would you like to submit from another branch, so you can keep your changes and we haven't got loads of these commits merged into our history?